### ISSUE TITLE
Checkbox border in Events->Configuration has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#863]
The following AI-windows mode(s) have this issue:
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![10 1](https://user-images.githubusercontent.com/22068434/88347228-248a6d80-ccff-11ea-8303-592e764f9be9.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
This is fixed by #943.

@iamrafan, this fix will be in release 1.1.1370.01 (will release October 2nd). Please regress and close once the build is out.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1380.01
